_posts/2020-09-16-uwbcode2020-announcement.md

```diff
@@ -12,6 +12,50 @@ exturl_uwbacm: https://uwbacm.com
 exturl_gwc: https://www.instagram.com/girlswhocode__uwb/?igshid=lmenj5wiuyo
 exturl_regisform: http://bit.ly/join-uwbcode20
 ---
+<!-- Display the countdown timer in an element -->
+<script>
+// Set the date we're counting down to
+var countDownDate = new Date("Oct 9, 2020 10:00:00 PDT").getTime();
+
+// Update the count down every 1 second
+var x = setInterval(function() {
+
+  // Get today's date and time
+  var now = new Date().getTime();
+
+  // Find the distance between now and the count down date
+  var distance = countDownDate - now;
+
+  // Time calculations for days, hours, minutes and seconds
+  var days = Math.floor(distance / (1000 * 60 * 60 * 24));
+  var hours = Math.floor((distance % (1000 * 60 * 60 * 24)) / (1000 * 60 * 60));
+  var minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60));
+  var seconds = Math.floor((distance % (1000 * 60)) / 1000);
+
+  // Display the result in the element with id="demo"
+  document.getElementById("clk").innerHTML = days + " days : " + hours + " hours : "
+  + minutes + " minutes : " + seconds + " seconds ";
+
+  // If the count down is finished, write some text 
+  if (distance < 0) {
+    clearInterval(x);
+    document.getElementById("clk").innerHTML = "CONTEST ENDED!";
```

Comment:
should this be "contest started"?

Reply:
you're absolutely right. countdown ended, contest started 😄